quickstart/csharp/unity/text-to-speech/Assets/Scripts/HelloWorld.cs

```diff
@@ -93,17 +126,22 @@ void Start()
             speakButton.onClick.AddListener(ButtonClick);
 
             // Creates an instance of a speech config with specified subscription key and service region.
-            // Replace with your own subscription key and service region (e.g., "westus").
-            speechConfig = SpeechConfig.FromSubscription("YourSubscriptionKey", "YourServiceRegion");
+            speechConfig = SpeechConfig.FromSubscription(SubscriptionKey, Region);
 
-            // The default format is Riff16Khz16BitMonoPcm.
```

Comment:
The comments are not clear for me. I suggest

// The default format is a RIFF format with header. We are playing the audio in memory as audio clip, which doesn't require RIFF header.
// So we need to set the format to Raw24Khz16BitMonoPcm (using 24KHz for better quality).